### PR TITLE
Add validation

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -18,10 +18,16 @@ variable "role" {
...
   default     = null
 }
+
 variable "members" {
   type        = set(string)
   description = "(Optional) Identities that will be granted the privilege in role. Each entry can have one of the following values: 'user:{emailid}', 'serviceAccount:{emailid}', 'group:{emailid}', 'domain:{domain}'."
   default     = []
+
+  validation {
+    condition     = alltrue([for m in var.members : can(regex("^(user|serviceAccount|group|domain):(.+)", m))])
```
Comment (mariux) [inline variables.tf:28]:
```suggestion
    condition     = alltrue([for m in var.members : can(regex("^(user|serviceAccount|group|domain):", m))])
```

**Review by mariux — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (soerenmartius) [reply variables.tf]: resolved in 76e41f03

**Review by kassianh — APPROVED** [review]
lgtm

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**